### ISSUE TITLE
Can't build version 0.16.0 with Rust 1.58: undeclared crate or module `log`

### ISSUE THREAD

**Issue by kauron** [twistedfall/libstrophe#6]
Hi there!

I've found a problem while trying to compile a project that uses your library. The build process crashes with the following error:
```
error[E0433]: failed to resolve: use of undeclared crate or module `log`
   --> $HOME/.cargo/registry/src/github.com-1ecc6299db9ec823/libstrophe-0.16.0/src/connection.rs:522.6
    |
522 |       if log::log_enabled!(log::Level::Debug) {
    |          ^^^ use of undeclared crate or module `log`
```
When I reported the issue to that project (see valeriansaliou/vigil/issues/108), one suggestion I received was to use Rust <= 1.21, but I'm on a rolling release Linux distro and even on more stable distros, Rust is already on version 1.34.

I'm not too familiar with Rust, so I'm afraid I don't have any proposals or patches.

**Comment by twistedfall** [comment]
Thanks for the report, I'll check it out!

**Comment by twistedfall** [comment]
It should be fixed in v0.16.1
